### PR TITLE
Minor ovs maintenance

### PR DESCRIPTION
**Pull request by ffoulkes**
- Supply a missing #include.
- Update Intel copyright notices.

### CODE REVIEW

**Review by 5abeel — APPROVED** [review]
LGTM

File: lib/mac-learning.c [hunk 1]
```diff
@@ -24,6 +24,7 @@
 #include "coverage.h"
 #include "hash.h"
 #include "openvswitch/list.h"
+#include "openvswitch/ovs-p4rt.h"
```
Comment (nupuruttarwar) [inline lib/mac-learning.c:27]: line 35 already includes this file and also it should be wrapped under P4OVS supported checks?

Comment (ffoulkes) [reply lib/mac-learning.c:27]: Nuts! You're right.

**Comment by ffoulkes** [comment]
This PR turns out to be completely unnecessary. I must have been asleep at the switch.

**Pull request closed without merging**